client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.ts

```diff
@@ -182,7 +182,7 @@ export class ActiveWildfireMapComponent implements OnInit, AfterViewInit {
   }});
 }
   
-  panToLocation(long, lat) {
+  panToLocation(long, lat, noZoom?) {
```

Comment:
Would it be more straightforward to define this as a default? `zoom: number | null = 10`
The boolean usage in panToFeature seems unecessary